.github/workflows/ci.yml

```diff
@@ -54,19 +55,16 @@ jobs:
       matrix:
         try-scenario:
           - ember-lts-3.28
-          - ember-release
-          - ember-beta
-          - ember-canary
```

Comment:
these fail due to the latest ember versions, currently this addon is not compatible with ember 5, we will need to make additional changes for it to work. Since we use ember 4 in boundary-ui, it made sense to comment these out and deal with this when we upgrade to ember 5